openmetadata-ui/src/main/resources/ui/src/components/Users/UserProfileIcon/UserProfileIcon.component.tsx

```diff
@@ -283,12 +283,13 @@ export const UserProfileIcon = () => {
 
   return (
     <Dropdown
+      data-testid="dropdown-profile"
       menu={{
         items,
         defaultOpenKeys: ['personas', 'roles', 'inheritedRoles', 'teams'],
       }}
       trigger={['click']}>
-      <div className="app-user-icon">
+      <div className="app-user-icon" data-testid="dropdown-profile">
```

Comment:
Wouldn't this be a duplicate data test id?

Reply:
It's not working for <Dropdown> I will remove it in next